gu-hub-api.yaml

```
@@ -0,0 +1,560 @@
+swagger: '2.0'
+info:
+  title: Golem unlimited low level hub API
+  description: API description in Markdown.
+  version: 1.0.0
+host: '127.0.0.1:61622'
+tags:
+  - name: hub-session
+    description: HUB Session managment.
+  - name: peer-info
+    description: Unlimited network peer info managment.
+  - name: peer-session
+  - name: lob-man
+    description: Large objects managment
+
+schemes:
+  - http
+security:
+  - gu-api-key: []
+  - systemName: []
+paths:
+  /peer:
+    get:
+      tags:
+        - peer-info
+      operationId: listPeers
+      summary: Returns a list hub peers.
+      parameters:
+        - $ref: '#/parameters/offset'
+        - $ref: '#/parameters/limit'
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: OK
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/PeerInfo'
+  /peer/{nodeId}:
+    parameters:
+      - name: nodeId
```

you defined it in parameters, so a reference can be used
```suggestion
      - $ref: '#/parameters/offset'
```